packages/shared/src/styles/themes.scss

```diff
@@ -82,6 +82,8 @@
         --button-primary-hover: #{$color-red-5};
         --button-secondary-hover: #{$alpha-color-black-3};
         --button-tertiary-hover: #{$alpha-color-red-1};
+        --button-toggle-primary: #{$color-apple-blue};
```

Comment:
I think amam is adding these in his PR, let's merge his changes first

Reply:
@negarn sure, I will update accordingly.